RELEASE_NOTES.txt

```diff
@@ -1,6 +1,120 @@
 Apache Ignite Release Notes
 ===========================
 
+Apache Ignite In-Memory Distributed Database 2.14.0
+-----------------------------------------------------------
+
+(!) WARNINGS:
+* Removed LOCAL caches support.
+* Removed scalar module.
+
+Ignite:
+* Added CDC binary meta change events.
+* Added a snapshot create operation metrics.
+* Added a user info to the sql queries view.
+* Added cache change events for CDCCosumer.
+* Added cache metrics command for Control Script.
+* Added control.sh command that schedules index rebuild via the maintenance mode.
```

Comment:
Added a control.sh command that schedules index rebuild via the maintenance mode.